lib/reverse_sentence.rb

```diff
@@ -1,6 +1,35 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: o(n^2)
+# Space complexity: o(n)
```

Comment:
This is actually O(n) because  you never reverse the same section of the string more than twice.